app/views/log_events/index.html.haml

```diff
@@ -46,6 +46,7 @@
       %th= text("event_time")
       %th= text("event")
       %th= text("object")
+      %th= "Facility"
```

Comment:
We should use `Facility.model_name.human` here. Dartmouth always uses "Resource" instead.